.github/workflows/trigger-dependent-repo-update.yml

```diff
@@ -0,0 +1,29 @@
+name: Trigger dependent repo update workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate authentication token
+        id: gen-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.CVAT_BOT_APP_ID }}
+          private-key: ${{ secrets.CVAT_BOT_PRIVATE_KEY }}
+          repositories: ${{ secrets.CVAT_DEPENDENT_REPO }}
+
+      - name: Trigger private repository workflow
+        env:
+          GH_TOKEN: "${{ steps.gen-token.outputs.token }}"
+        run: |
+          gh workflow run \
+            update-dependency.yml \
+            --repo=cvat-ai/${{ secrets.CVAT_DEPENDENT_REPO }} \
```

Comment:
I would suggest to define the [`GH_REPO`](https://cli.github.com/manual/gh_help_environment) environment variable instead, to avoid passing secrets on the command line.